pkg/retriever/retriever.go

```diff
@@ -161,8 +161,9 @@ func (retriever *Retriever) Retrieve(
 		eventsCB,
 	)
 
+	urlPath, _ := request.GetUrlPath()
```

Comment:
in #282 I added `Duplicates bool` to `Request` so I could get an etag; we don't have `dups` as a query parameter but maybe we should introduce one so we can put it here if it's not default?

Reply:
I guess we'd have to have a conversation wrt spec to introduce it, so maybe we tack it on here manually?